src/js/facility-locator/components/SearchControls.jsx

```diff
@@ -18,11 +15,69 @@ class SearchControls extends Component {
     this.state = {
       facilityDropdownActive: false,
       serviceDropdownActive: false,
+      facilityDropdownFocused: false,
+      serviceDropdownFocused: false,
+      focusedFacilityIndex: 0,
+      focusedServiceIndex: 0
     };
-
+    this.services = [];
+    this.facilities = [];
+    this.focusFacilityOption = this.focusFacilityOption.bind(this);
+    this.focusServiceOption = this.focusServiceOption.bind(this);
+    this.goToFacilityDropdown = this.goToFacilityDropdown(this);
+    this.handleFacilityFilterSelect = this.handleFacilityFilterSelect.bind(this);
+    this.handleServiceFilterSelect = this.handleServiceFilterSelect.bind(this);
+    this.isSelectedFacility = this.isSelectedFacility.bind(this);
+    this.isFocusedFacilityIndex = this.isFocusedFacilityIndex.bind(this);
+    this.navigateFacilityDropdown = this.navigateFacilityDropdown.bind(this);
+    this.navigateServiceDropdown = this.navigateServiceDropdown.bind(this);
     this.toggleFacilityDropdown = this.toggleFacilityDropdown.bind(this);
     this.toggleServiceDropdown = this.toggleServiceDropdown.bind(this);
-    this.handleFacilityFilterSelect = this.handleFacilityFilterSelect.bind(this);
+    this.setFacilityDropdownFocus = this.setFacilityDropdownFocus.bind(this);
+    this.setServiceDropdownFocus = this.setServiceDropdownFocus.bind(this);
+  }
+
+  setFacilityDropdownFocus(newState) {
+    const dropdownState = false ? !this.state.facilityDropdownActive : this.state.facilityDropdownActive;
```

Comment:
Is this supposed to be `newState` instead of `false`? Also, are the focused and active states ever different?

Reply:
Good point, in fact, this logic is no longer necessary due to the focus update you mentioned below, and it has been removed.